draft-ietf-gnap-core-protocol.md

```diff
@@ -173,27 +173,27 @@ Legend
 ~~~
 
 Authorization Server (AS)
-: server that grants delegated privileges to a particular instance of client software in the form of access tokens or other information (such as subject information). 
+: server that grants delegated privileges to a particular instance of client software in the form of access tokens or other information (such as subject information).
 
 Client
-: application operated by an end-user that consumes resources from one or several RSs, possibly requiring access privileges from one or several ASs. 
+: application operated by an end-user that consumes resources from one or several RSs, possibly requiring access privileges from one or several ASs.
 
     Example: a client can be a mobile application, a web application, etc.
-    
```

Comment:
We should be careful of lines like this -- whitespace-only lines have semantic meaning in certain context in Markdown (particularly in list and block continuation). So do two spaces at the end of a line (linebreak without paragraph break). Is there a way to cover these odd cases by the editorconfig, which otherwise looks good?